nucypher/config/base.py

```diff
@@ -824,13 +836,20 @@ def write_keystore(
         password: str,
         key_material: Optional[bytes] = None,
         interactive: bool = True,
+        with_mnemonic: bool = False,
```

Comment:
Is there a conflict between `with_mnemonic` and  `key_material` i.e. if `key_material` is not `None` then `with_mnemoic` should be `False`, and the opposite i.e. if `with_mnemonic` is True then `key_material` should be `None`. Do we check that this is the case for the parameters somewhere in the call stack? I may have missed it.

Reply:
Yes - it's a good point.  I'll make an update.

Reply:
✔️